packages/test/local-server-stress-tests/src/baseModel.ts

```diff
@@ -69,6 +71,25 @@ export function makeGenerator<T extends BaseOperation>(
 			}),
 			5,
 		],
+		[
+			async () => ({
+				type: "enterStagingMode",
+			}),
+			5,
+			(state) =>
+				!state.client.entryPoint.inStagingMode() &&
+				state.client.container.attachState !== AttachState.Detached,
+		],
+		[
+			async ({ random }) => ({
+				type: "exitStagingMode",
+				commit: random.bool(),
+			}),
+			25,
```

Comment:
I wonder if we want to make this lower to get more changes in Staging Mode. At least in the minimized result JSON, it's often only 1 or 2 operations done in there.

Reply:
i actually think this work pretty well. we hit issue across all major op producing subsystem, attach, blobs, dds. i'm also not sure more ops is better.

something i've thought about is how do we validate the stress test itself. an idea i've had is we allow some kind of custom sampling/aggregation based on operations, and then have a validation phase at the end. this would allow tracking and checking anything. it might be a bit brittle, so need to be careful with it.